simulation_ws/src/deepracer_simulation/package.xml

```diff
@@ -8,26 +8,29 @@
   <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
   <license>Apache2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <depend>gazebo_ros</depend>
+  <depend>rclpy</depend>
+  <depend>gazebo_plugins</depend>
+  <depend>gazebo_ros_pkgs</depend>
+  <exec_depend>gazebo</exec_depend>
 
-  <build_depend>ackermann_msgs</build_depend>
+  <!-- TODO build_depend>ackermann_msgs</build_depend>
```

Comment:
Why are these commented out?

Reply:
[WIP] Right now this functionality is in progress so commented.